example/android/app/build.gradle

```diff
@@ -32,6 +32,8 @@ apply plugin: 'kotlin-kapt'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
+    namespace 'com.flutterandies.photo_manager_example'
```

Comment:
Code here needs to resolve conflicts because of #961.

Reply:
Conflict resolved.